Source/core/resources/fuels.json

```diff
@@ -0,0 +1,58 @@
+{  
+  "Diesel":{  
+    "langKey":"fuelDiesel",
+    "measuredBy":"v",
+    "CO2Emission":"2614"
+  },
+  "Gasoline":{  
+    "langKey":"fuelGasoline",
+    "measuredBy":"v",
+    "CO2Emission":"2328"
```

Comment:
Please include, as comments, details of how these values were computed, like we had before... (see below)